src/amo/containers/App.js

```diff
@@ -77,7 +85,7 @@ export class AppBase extends React.Component {
           SearchFormComponent={SearchForm} isHomePage={isHomePage} location={location}
           query={query} ref={(ref) => { this.mastHead = ref; }} />
         <div className="App-content">
-          {children}
+          {errorPage.error ? 'errorContent' : children}
```

Comment:
That's pretty much what we already have: https://github.com/mozilla/addons-frontend/blob/master/src/core/resourceErrors/decorator.js#L27 I know you are trying to get something simple working first as a proof though.